proto/tendermint/types/types.proto

```diff
@@ -102,6 +102,12 @@ message Vote {
   bytes validator_address = 6;
   int32 validator_index   = 7;
   bytes signature         = 8;
+  VoteExtension vote_extension = 9;
+}
+
+message VoteExtension {
```

Comment:
Lets add a comment saying something to the extent of:

`VoteExtension` is a field returned by the application via ABCI, and must be gossiped along with the votes.